src/main/java/com/yurjinia/common/s3/configuration/S3Configuration.java

```diff
@@ -31,8 +28,8 @@ public class S3Configuration {
 
     @Bean
     public S3Client s3Client() {
-        accessKeyId = environmentService.getEnv(accessKeyId);
-        secretAccessKey = environmentService.getEnv(secretAccessKey);
+        /*accessKeyId = environmentService.getEnv(accessKeyId);
```

Comment:
Add ToDo with explanation why the code is commented. Otherwise if it's not needed - then delete it.

Also remove redundant spaces between @Value field.